runtime/src/main/java/org/corfudb/security/tls/TlsUtils.java

```diff
@@ -151,21 +153,35 @@ public Path getCertStore() {
 
         @AllArgsConstructor
         @Getter
+        @ToString
         class TrustStoreConfig implements CertStoreConfig {
+            public static final Path DEFAULT_DISABLE_CERT_EXPIRY_CHECK_FILE = Paths.get(
+                    "/", "config", "corfu", "DISABLE_CERT_EXPIRY_CHECK"
+            );
```

Comment:
Rather than hard-coding this, can't you get the root data directory from the `-l` parameter? This approach won't work for non-config Corfu.